packages/react/src/components/button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Could you remove the `Button.js`-component from this directory?

Reply:
Good spot! Removed.